frontend/test/metabase/scenarios/question/reproductions/12445-mysql-cc-apply-substring.cy.spec.js

```diff
@@ -0,0 +1,38 @@
+import { restore, withDatabase } from "__support__/e2e/cypress";
+
+const CC_NAME = "Abbr";
+
+describe.skip("issue 12445", () => {
+  beforeEach(() => {
+    restore("mysql-8");
+    cy.signInAsAdmin();
+  });
+
+  it("should correctly apply substring for a custom column (metabase#12445)", () => {
+    withDatabase(2, ({ PEOPLE, PEOPLE_ID }) => {
```

Comment:
Could the ID reference be a variable? Just makes it easier to read.

And I'm quite sure that `PEOPLE, PEOPLE_ID` etc variables will fail, since they are likely referencing H2's IDs. I might have missed an index file somewhere or some auto-magic.

Reply:
`withDatabase` is the magic you're talking about. This is the same as when we were using `withSampleDataset()` before and then desctructure table and field IDs.